index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+        <main id="main-doc">
+                  <h1>Agile and waterfall approaches</h1d>
+            <section1 id="#" class="main-section">
+              <h2>Agile and waterfall approaches</h2>The documentation types that the team produces and
+              its scope depending on the software development approach that was chosen. There are two main ones: agile and waterfall. Each is unique in terms of accompanying documentation.
+            </section1>
+
+            <section2 id="#" class="main-section">
+              <h2><h2>tages of the projesct.</h2></h2>The Waterfall approach is a linear method with distinct goals
+              for each development phase. Teams that use waterfall spend a</section2 id="#">
+              <section3 id="#" class="main-section">
+                <h2><h2>tages of the projesct.</h2>
+              </h2>reasonable amount of time on product planning in the early
+              </section3> 
+
+              <section4 id="#"  class="main-section">
```

Comment:
- [ ] Try using section tags rather than adding numbers to them

Reply:
- [ ] Kindly make your request specific and straight to the point. For instance you said 'Try using section tags rather than adding numbers to them'. Being more specific will help avoid review loops. :wink: